tools/tcpconnlat.py

```diff
@@ -206,23 +206,23 @@ def positive_float(val):
 class Data_ipv4(ct.Structure):
     _fields_ = [
         ("ts_us", ct.c_ulonglong),
-        ("pid", ct.c_ulonglong),
+        ("pid", ct.c_uint),
         ("saddr", ct.c_uint),
         ("daddr", ct.c_uint),
         ("ip", ct.c_ulonglong),
-        ("dport", ct.c_ulonglong),
+        ("dport", ct.c_uint),
```

Comment:
dport should be ct.c_ushort?

Reply:
yeah,ct.c_uint also works,  had changed to ct.c_ushort. Thanks